cards/acute-limb-ischemia/card.md

```diff
@@ -34,16 +34,16 @@ categories:
 - **Neurological:** Check for sensory and motor loss
 - **Vascular:** Grade peripheral pulses
 
-- 0 = No Doppler signal
-- 1 = Reduced
-- 2 = Normal
-- 3 = Increased
-- 4 = Bounding
+    - 0 = No Doppler signal
```

Comment:
Actually, I think they do need an extra indent. The first order bullet is organ system (neuro, extremity, vascular). The reason for the further indent is that these pulse go under Vascular. 

Can we do such an indent without creating errors in the Travis / build?